dotcom-rendering/src/layouts/DecideLayout.tsx

```diff
@@ -114,6 +115,14 @@ const DecideLayoutWeb = ({
 							format={format}
 						/>
 					);
+				case ArticleDesign.Picture:
```

Comment:
Do we want to test this - or are you happy to shoot it off to 100% of readers, which might make fielding bugs a bit more stressful?